pkg/ctrl/server/process.go

```diff
@@ -69,3 +70,35 @@ func (r *Resolver) GetPidFromPS(ctx context.Context, pod *v1.Pod) ([]*model.Proc
 	}
 	return processes, nil
 }
+
+// GetPidFromPS returns pid-command pairs
+func (r *Resolver) killProcess(ctx context.Context, pod *v1.Pod, pids []string) ([]*model.KillProcessResult, error) {
+	pidSet := make(map[string]bool)
```

Comment:
Can you add some comment about where theres pids come from?

Reply:
Added